### ISSUE TITLE
dump engine stopped working after upgrading to 0.18.12

### ISSUE THREAD

**Issue by denyska** [tilt-dev/tilt#4316]
### Expected Behavior
`tilt dump engine` should produce dump of internal state. We use this json to query status of the services.

### Current Behavior

```
 ~/Downloads/tilt.0.18.12.mac.x86_64/tilt dump engine
dump engine: Could not decode: invalid character ':' looking for beginning of object key string
```

however it works just fine when I move back to `0.18.11`

```
➜  ~/Downloads/tilt.0.18.11.mac.x86_64/tilt dump engine
{
  "AnalyticsEnvOpt": 0,
  "AnalyticsNudgeSurfaced": false,
  "AnalyticsTiltfileOpt": 0,
  "AnalyticsUserOpt": 2,
  "CloudAddress": "cloud.tilt.dev",
  "CloudStatus": {
    "TeamName": "",
    "TokenKnownUnregistered": false,
    "Username": "denyska",
    "WaitingForStatusPostRegistration": false
  },
  "CompletedBuildCount": 0,
  "ConfigFiles": [
    "/Users/denyska/Projects/myrepo/Tiltfile"
  ],
  "CurrentlyBuilding": {},
  "DockerPruneSettings": {
    "Enabled": false,
    "Interval": 0,
    "KeepRecent": 0,
    "MaxAge": 0,
    "NumBuilds": 0
  },
  "EngineMode": {
....
```

### Steps to Reproduce

1.
2.
3.
4.

#### `tilt doctor` Output
```
~/Downloads/tilt.0.18.12.mac.x86_64/tilt  doctor
Tilt: v0.18.12, built 2021-03-09
System: darwin-amd64
---
Docker (cluster)
- Host: tcp://192.168.64.48:2376
- Version: 1.41
- Builder: 2
---
Docker (local)
- Host: [default]
- Version: 1.41
- Builder: 2
---
Kubernetes
- Env: minikube
- Context: minikube
- Cluster Name: minikube
- Namespace: default
- Container Runtime: docker
- Version: v1.20.2
- Cluster Local Registry: none
---
Thanks for seeing the Tilt Doctor!
Please send the info above when filing bug reports. 💗

...
```

**Comment by milas** [comment]
Thanks for the report! This has been fixed and will be in the next Tilt release (sometime this week). I'll update here once it's out.

**Comment by nicks** [comment]
This is fixed in v0.19!
